### DISCUSSION TITLE
how to add custom net in metamask for foundry course

### DISCUSSION THREAD

**Discussion by DevinBeu** [Cyfrin/foundry-full-course-cu#2984] [General]
metamask looks totally different now than it did at the time of the video. I cant find where to add a custom network?
![metafoundry](https://github.com/user-attachments/assets/e837ed9f-17b9-45f4-b0e5-c7d02c48a335)

**Comment by NikolayPIvanov** [comment]
I do not think you can do this from portfolio.metamask, I suggest using the browser extension. From there on the top left corner you can click on the networks and add a new custom one.

**Reply by DevinBeu** [reply]
thank you sir!

**Comment by EngrPips** [comment]
Great heads-up.

**Comment by heheboii11** [comment]
Hey @DevinBeum, you can add a custom network as shown in the image, just click on the ethereum icon on the left corner
<img width="359" alt="Screenshot 2024-12-05 at 12 40 38 PM" src="https://github.com/user-attachments/assets/dd9e2c06-1b48-4b29-b655-0d2a5a2b0991">
<img width="359" alt="Screenshot 2024-12-05 at 12 41 01 PM" src="https://github.com/user-attachments/assets/3212aacf-ceeb-4bf1-a1d0-2502e94eec5e">
